dennerparreiras/RequestMocker
# Enhance build process and update .gitignore for .NET Framework 4.7.2 project

## Enhance build process and update .gitignore for .NET Framework 4.7.2 project

## Description
This pull request addresses the need for explicit .NET Framework 4.7.2 support in our project and updates the .gitignore file for better project hygiene. 

We added the `Microsoft.NETFramework.ReferenceAssemblies` package to our .csproj file to ensure that the correct .NET Framework version is used during our GitHub actions build process. This change should help prevent build failures due to the absence of the required .NET Framework version.

Additionally, we've updated our .gitignore file to ignore `bin/`, `obj/`, `lastToken`, and `requestsLog.txt`. This will prevent unnecessary files from being tracked in our repository.

## Areas of the application affected
- .csproj file
- .gitignore file
- Build process

## How to test
1. Merge the PR.
2. Trigger a build in GitHub Actions.
3. Ensure that the build process completes without any errors related to .NET Framework 4.7.2.
4. Make changes that would affect the now ignored files and directories (`bin/`, `obj/`, `lastToken`, and `requestsLog.txt`). Ensure these changes are not tracked by Git.

## Screenshots
[N/A]
